CHANGELOG.md

```diff
@@ -29,6 +29,12 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 - A profile created with the `--disable-ssl-errors` flag will now correctly not verify SSL certs when making requests. A warning message is printed
     each time the CLI is run with a profile configured this way, as it is not recommended.
 
+- Below `search` subcommands accept argument `--format/-f` to display result in formats `csv`, `table`, `json`, `formatted-json`:
```

Comment:
Didn't the default output change for a lot of commands as well? We should also mention that... What before returned a JSON blob, now returns a terminal table without using the proper flag.

Reply:
Yea this should change. Wondering, do we want to go ahead with the breaking change or shall we default it to JSON blob, as earlier? 
Is it possible to get a confirmation on whether the default format can be changed? i.e there isn't any automation around this? or just declaring that this is a breaking change would suffice, any one who updates to latest version will do the needful.

Reply:
The default output format should change to table. Calling this out as a breaking change should suffice in this case.

Reply:
Done.